fork-client/build.gradle

```diff
@@ -0,0 +1,10 @@
+apply plugin: 'java-library'
+apply from: '../build-scripts/gradle-mvn-push.gradle'
+dependencies {
```

Comment:
This project has no dependencies, so can we remove this generated line?

Reply:
👍